xml/slurm.xml

```diff
@@ -1286,7 +1286,7 @@ SlurmdTimeout=3600</screen>
  </sect2>
  </sect1>
  <sect1 xml:id="sec-slurm-additional-res">
-  <title>Additional Ressources</title>
+  <title>Additional ressources</title>
   <sect2 xml:id="sec-slurm-faq">
    <title>Frequently asked questions</title>
```

Comment:
```suggestion
   <title>Frequently-asked questions</title>
```

Reply:
From all I can see, adding a hyphen here is a rather unusual. Otherwise, this feels potato/tomahto to me.

Reply:
There *is* a rule, and this breaks it, but it's not vitally important. If you prefer to leave it, no problem.